frontend/src/metabase-types/api/card.ts

```diff
@@ -92,45 +110,188 @@ export type TableColumnOrderSetting = {
+  "graph.y_axis.axis_enabled"?: boolean;
+  "graph.y_axis.labels_enabled"?: boolean;
+  "graph.y_axis.max"?: number;
+  "graph.y_axis.min"?: number;
+  "graph.y_axis.scale"?: "linear" | "pow" | "log";
+  "graph.y_axis.title_text"?: string | null;
+  "line.interpolate"?: "linear" | "cardinal" | "step-after";
+  "line.marker_enabled"?: null | boolean;
+  "line.missing"?: "zero" | "none" | "interpolate";
+  "map.center_latitude"?: number;
+  "map.center_longitude"?: number;
+  "map.colors"?: string[];
+  "map.dimension"?: string;
+  "map.heat.blur"?: number;
+  "map.heat.max-zoom"?: number;
+  "map.heat.min-opacity"?: number;
+  "map.heat.radius"?: number;
+  "map.latitude_column"?: string;
+  "map.longitude_column"?: string;
+  "map.metric"?: string;
+  "map.metric_column"?: string;
+  "map.pin_type"?: "tiles" | "markers" | "heat" | "grid";
+  "map.region"?: "us_states" | "world_countries" | string;
+  "map.type"?: "region" | "pin" | "heat" | "grid";
+  "map.zoom"?: number;
+  "pie._colors"?: string[];
+  "pie._dimensionIndex"?: number;
+  "pie._dimensionTitles"?: string[] | null;
+  "pie._dimensionValues"?: string[] | null;
+  "pie._metricIndex"?: number;
+  "pie.colors"?: string[];
+  "pie.dimension"?: string;
+  "pie.metric"?: string;
+  "pie.percent_visibility"?: "off" | "legend" | "inside";
+  "pie.show_legend"?: boolean;
+  "pie.show_total"?: boolean;
+  "pie.slice_threshold"?: number;
+  "pivot.show_column_totals"?: boolean;
+  "pivot.show_row_totals"?: boolean;
   "pivot_table.collapsed_rows"?: PivotTableCollapsedRowsSetting;
+  "pivot_table.column_show_totals"?: boolean;
+  "pivot_table.column_split"?: PivotSetting;
+  "pivot_table.column_widths"?: HeaderWidthType;
+  "progress.color"?: string;
+  "progress.goal"?: number;
+  "scalar.field"?: string;
+  "scalar.switch_positive_negative"?: boolean;
+  "scalar.decimals"?: number; // legacy
+  "scalar.locale"?: null | string; // legacy
+  "scalar.prefix"?: string; // legacy
+  "scalar.scale"?: number; // legacy
+  "scalar.suffix"?: string; // legacy
+  "scatter.bubble"?: string | null;
+  "stackable.stack_display"?: "area" | "bar";
+  "stackable.stack_type"?: "stacked" | "normalized" | null;
+  "stackable.stacked"?: VisualizationSettings["stackable.stack_type"]; // legacy
+  "table._cell_background_getter"?: (
+    value: unknown,
+    rowIndex: number,
+    colName: string,
+  ) => string;
+  "table.cell_column"?: string;
+  "table.column_formatting"?: ColumnFormattingSetting[];
+  "table.column_widths"?: number[];
+  "table.columns"?: TableColumnOrderSetting[];
+  "table.pivot"?: boolean;
+  "table.pivot_column"?: string;
+  "text.align_horizontal"?: "left" | "center" | "right";
+  "text.align_vertical"?: "top" | "middle" | "bottom";
+  "waterfall.decrease_color"?: string;
+  "waterfall.increase_color"?: string;
+  "waterfall.show_total"?: boolean;
+  "waterfall.total_color"?: string;
+
+  // Nested settings
+  column?: (column: DatasetColumn) => VisualizationColumnSettings;
```

Comment:
This is the api type that describes json we receive from the BE so it does not have functions. That is why there was `VisualizationSettings` and `ComputedVisualizationSettings`